src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -0,0 +1,80 @@
+package seedu.addressbook.commands;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+import java.util.*;
```

Comment:
Instead of importing the entire package, import only the classes you need. If its happening automatically, you can change it at "code style -> Java -> imports" in IntelliJ